### PR TITLE
PEP8 fixes for corpora

### PR DESCRIPTION
**Pull request by souravsingh**
Fixes a small part of #965

### CODE REVIEW

File: gensim/corpora/lowcorpus.py [hunk 1]
```diff
@@ -119,7 +119,7 @@ def line2doc(self, line):
                     marker.add(word)
             # construct a list of (wordIndex, wordFrequency) 2-tuples
             doc = list(zip(map(self.word2id.get, use_words),
-                      map(words.count, use_words)))
+                           map(words.count, use_words)))
```
Comment (piskvorky) [inline gensim/corpora/lowcorpus.py]: No vertical indent in gensim; use hanging indent in all cases (& no statement on first line).

File: gensim/corpora/lowcorpus.py [hunk 2]
```diff
@@ -166,8 +166,8 @@ def save_corpus(fname, corpus, id2word=None, metadata=False):
 
         if truncated:
             logger.warning("List-of-words format can only save vectors with "
-                            "integer elements; %i float entries were truncated to integer value" %
-                            truncated)
+                           "integer elements; %i float entries were truncated to integer value" %
```
Comment (piskvorky) [inline gensim/corpora/lowcorpus.py]: Dtto (please use hanging indent consistently).

Also, we want to be passing logging message string parameters as arguments, not using them to format the string directly (wasted effort if message doesn't pass logging filters): `logger.fnc("%s", x)`, not `logger.fnc("%s" % x)`.

File: gensim/corpora/sharded_corpus.py [hunk 3]
```diff
@@ -236,7 +236,7 @@ def __init__(self, output_prefix, corpus, dim=None,
                                      # corresponds to index 0 of current shard
 
         logger.info('Initializing sharded corpus with prefix '
-                     '{0}'.format(output_prefix))
```
Comment (piskvorky) [inline gensim/corpora/sharded_corpus.py]: Hanging indent.

File: gensim/corpora/sharded_corpus.py [hunk 4]
```diff
@@ -245,7 +245,7 @@ def __init__(self, output_prefix, corpus, dim=None,
             # and retain information about how the corpus
             # was serialized.
             logger.info('Saving ShardedCorpus object to '
-                         '{0}'.format(self.output_prefix))
+                        '{0}'.format(self.output_prefix))
```
Comment (piskvorky) [inline gensim/corpora/sharded_corpus.py]: Hanging indent.

File: gensim/corpora/sharded_corpus.py [hunk 5]
```diff
@@ -262,12 +262,12 @@ def init_shards(self, output_prefix, corpus, shardsize=4096, dtype=_default_dtyp
         if proposed_dim != self.dim:
             if self.dim is None:
                 logger.info('Deriving dataset dimension from corpus: '
-                             '{0}'.format(proposed_dim))
+                            '{0}'.format(proposed_dim))
```
Comment (piskvorky) [inline gensim/corpora/sharded_corpus.py]: Hanging indent. Last comment -- please fix all.

**Comment by piskvorky** [comment]
What's happening with these pull request? I write a review and then they disappear again, closed without merging...?

**Comment by souravsingh** [comment]
Sorry for closing this PR. I was trying to create separate PR for corpora and topic_coherence in different branches, but somehow the changes for both corpora and topic_coherence got into the PR #1002

**Comment by piskvorky** [comment]
My review comments here stand; please address them in the other PRs.

**Pull request closed without merging**